### PR TITLE
As of OCP 4.1.2 elasticsearch urls need to be fully qualified

### PR DESCRIPTION
**Pull request by kevinearls**
Signed-off-by: Kevin Earls <kearls@redhat.com>

### CODE REVIEW

**Comment by kevinearls** [comment]
@objectiser @pavolloffay @jpkrohling Please review

File: test/e2e/self_provisioned_elasticsearch_test.go [hunk 1]
```diff
@@ -112,6 +116,9 @@ func getJaegerSimpleProd() *v1.Jaeger {
 			Strategy: "production",
 			Storage: v1.JaegerStorageSpec{
 				Type: "elasticsearch",
+				Options: v1.NewOptions(map[string]interface{}{
```
Comment (objectiser) [inline test/e2e/self_provisioned_elasticsearch_test.go]: @jpkrohling @pavolloffay Hasn't this changed the nature of the test, from being self-provisioned?

Comment (jpkrohling) [reply test/e2e/self_provisioned_elasticsearch_test.go]: Yes, it has, good catch!

Comment (pavolloffay) [reply test/e2e/self_provisioned_elasticsearch_test.go]: +1 This should be reverted then, otherwise it's not testing self-provisioned  ES. @kevinearls ?

Comment (kevinearls) [reply test/e2e/self_provisioned_elasticsearch_test.go]: @pavolloffay I'll either revert it or apply another fix by the end of the day.

**Pull request merged**